lms/djangoapps/discussion/rest_api/urls.py

```diff
@@ -56,5 +56,6 @@
         CourseTopicsView.as_view(),
         name="course_topics"
     ),
+    path("^v1/users/", CourseUsersView.as_view(), name="discussion_course_users"),
```

Comment:
@xitij2000 don't you think we can improve this URL so that it's self intuitive? e.g. `v1/courses/{course-id}/activity_stats/`?

Reply:
`/api/discussion/v1/courses/course-v1:ExampleX+Subject101+2015/activity_stats/?order_by=activity`

Reply:
I've updated the url as suggested.